### ISSUE TITLE
Kernel panic and unable to start afterwards

### ISSUE THREAD

**Issue by johanneswuerbach** [docker/for-mac#1068]
### Expected behavior

No kernel panic and docker startings afterwards

### Actual behavior

Kernel panic and docker is unable to start, reset to factory defaults is blocked by the hyperkit failed to start screen.

### Information

`ID: 2B34FEA4-00EB-460D-8CB9-BD54BC14DE1F`

```
Docker for Mac: version: 1.13.0-rc4-beta34 (264590153)
OS X: version 10.12.2 (build: 16C67)
logs: /tmp/2B34FEA4-00EB-460D-8CB9-BD54BC14DE1F/20161221-193312.tar.gz
failure: com.docker.slirp is not running
[OK]     vmnetd
[OK]     dns
[ERROR]  driver.amd64-linux
         com.docker.driver.amd64-linux -db is not running
[OK]     virtualization VT-X
[OK]     app
[ERROR]  moby
         /Users/johannes/Library/Containers/com.docker.docker/Data/com.docker.driver.amd64-linux/console-ring does not exist
[OK]     system
[OK]     moby-syslog
[ERROR]  db
         Connection refused (ECONNREFUSED) connecting to /Users/johannes/Library/Containers/com.docker.docker/Data/s40: check if service is running
         com.docker.db is not running
[OK]     env
[OK]     virtualization kern.hv_support
[ERROR]  slirp
         Unexpected error ((Failure
  "Error connecting socket to 9p endpoint unix:/Users/johannes/Library/Containers/com.docker.docker/Data/s51: Unix.Unix_error(Unix.ECONNREFUSED, \"connect\", \"\")")) connecting to /Users/johannes/Library/Containers/com.docker.docker/Data/s51
         com.docker.slirp is not running
         slirp check failed with: (Failure
  "Error connecting socket to 9p endpoint unix:/Users/johannes/Library/Containers/com.docker.docker/Data/s52: Unix.Unix_error(Unix.ECONNREFUSED, \"connect\", \"\")")
[ERROR]  osxfs
         com.docker.osxfs is not running
[OK]     moby-console
[OK]     logs
[ERROR]  docker-cli
         Connection refused (ECONNREFUSED) connecting to /var/run/docker.sock: check if service is running
         Connection refused (ECONNREFUSED) connecting to /Users/johannes/Library/Containers/com.docker.docker/Data/s60: check if service is running
         docker ps failed
[OK]     menubar
[OK]     disk
```

### Steps to reproduce the behavior

Pulling and creating a relatively complex docker-compose environment, not really an idea what to look for.

**Comment by djs55** [comment]
As a workaround for the "Reset to factory defaults" being hidden, try
```
rm ~/Library/Containers/com.docker.docker/Data/com.docker.driver.amd64-linux/Docker.qcow2
```
and then restart the app.

I believe this is the same as #1071.

Docker for Mac beta 34+ added extra integrity checks for the qcow2 file on VM start. Your logs contain the error
```
block device raised exception: (Failure "Found a reference to cluster 312655 outside the file (max cluster 299164) from cluster 78.35912")
```
Unfortunately this means that -- sometime in the past -- two groups of sectors in the virtual disk have become mapped to the same physical sector. It's unsafe to write to the file in this state. Unfortunately the only option is to use "Whale menu" -> "Preferences..." -> "Reset" -> "Reset to factory defaults". Unfortunately your containers and images will need to be rebuilt.

I suspect this may have happened when we enabled "online compaction" feature to reclaim space in the qcow2 file. The online feature is now disabled so I believe the problem should be fixed.

I apologies for the inconvenience. By the way beta 34 hotfix 1 was just released -- it has a bug fix relating to importing containers from toolbox. I recommend running the hotfix if you plan to import from toolbox.

**Comment by johanneswuerbach** [comment]
Thanks a lot for the detailed response, I already factory reseted my installation. Would be nice if this would be an option on the Fatal Error error screen.

While building another largish app (using a mounted macOS folder) I received another kernel panic with 1.13.0-rc4-beta34.1:
Upload ID: B113B3D6-4AAF-4277-9C73-C7AAA2E35F5B

<details>
  <summary>Panic logs</summary>
   <p>

#### Today
```
Anonymous UUID:       6EC9C959-1935-64E3-89E8-47EFC08F4DDA

Thu Dec 22 18:08:17 2016

*** MCA Error Report ***
CPU Machine Check Architecture Error Dump (CPU: Intel(R) Core(TM) i7-5557U CPU @ 3.10GHz, CPUID: 0x306D4)
Core: 0 
  IA32_MC3_STATUS=0xBE00000000800400
  IA32_MC3_CTL=0x0
  IA32_MC3_ADDR=0xFFFFFFFF8134FFFD
  IA32_MC3_MISC=0xFFFFFFFF8134FFFD
Core: 0 
  IA32_MC4_STATUS=0xBE00000000800400
  IA32_MC4_CTL=0x0
  IA32_MC4_ADDR=0xFFFFFFFF8134FFFD
  IA32_MC4_MISC=0xFFFFFFFF8134FFFD
Core: 0 
  IA32_MC5_STATUS=0xEE0000000040110B
  IA32_MC5_CTL=0x0
  IA32_MC5_ADDR=0xFF7075C0
  IA32_MC5_MISC=0x3880000086
Core: 0 
  IA32_MC6_STATUS=0xEE0000000040110B
  IA32_MC6_CTL=0x0
  IA32_MC6_ADDR=0xFF70FBC0
  IA32_MC6_MISC=0x3880000086


*** Panic Report ***
Machine-check capabilities: 0x0000000001000c07
 family: 6 model: 61 stepping: 4 microcode: 33
 signature: 0x306d4
 Intel(R) Core(TM) i7-5557U CPU @ 3.10GHz
 7 error-reporting banks
Processor 2: IA32_MCG_STATUS: 0x0000000000000005
 IA32_MC4_STATUS(0x411): 0xbe00000000800400
 IA32_MC4_ADDR(0x412):   0x<ptr>
 IA32_MC4_MISC(0x413):   0x<ptr>
Processor 3: IA32_MCG_STATUS: 0x0000000000000005
 IA32_MC4_STATUS(0x411): 0xbe00000000800400
 IA32_MC4_ADDR(0x412):   0x<ptr>
 IA32_MC4_MISC(0x413):   0x<ptr>
panic(cpu 3 caller 0xffffff8010206a05): "Machine Check at 0xffffff8010219adb, registers:\n" "CR0: 0x000000008001003b, CR2: 0x000000c41ff7f900, CR3: 0x00000000138e0000, CR4: 0x00000000003627e0\n" "RAX: 0x0000000000000060, RBX: 0xffffff8010893740, RCX: 0x0000000000000001, RDX: 0x0000000000000000\n" "RSP: 0xffffff92214d3ee0, RBP: 0xffffff92214d3f20, RSI: 0x0000000000000019, RDI: 0xffffff8010808e10\n" "R8:  0x0000000000000004, R9:  0x0000000000000000, R10: 0x0000000000000003, R11: 0xffffff8053191d00\n" "R12: 0x0000000000000060, R13: 0x0000000000000003, R14: 0x000025f7cfaacfcd, R15: 0x0000000000000000\n" "RFL: 0x0000000000000046, RIP: 0xffffff8010219adb, CS:  0x0000000000000008, SS:  0x0000000000000010\n" "Error code: 0x0000000000000000\n"@/Library/Caches/com.apple.xbs/Sources/xnu/xnu-3789.31.2/osfmk/i386/trap_native.c:168
Backtrace (CPU 3), Frame : Return Address
0xffffff81ed7ade90 : 0xffffff80100f211c 
0xffffff81ed7adf10 : 0xffffff8010206a05 
0xffffff81ed7ae070 : 0xffffff80100a3fdf 
0xffffff92214d3f20 : 0xffffff8010207d00 
0xffffff92214d3f40 : 0xffffff801010e94d 
0xffffff92214d3f90 : 0xffffff801010ef30 
0xffffff92214d3fb0 : 0xffffff80100a0af7 

BSD process name corresponding to current thread: kernel_task

Mac OS version:
16C67

Kernel version:
Darwin Kernel Version 16.3.0: Thu Nov 17 20:23:58 PST 2016; root:xnu-3789.31.2~1/RELEASE_X86_64
Kernel UUID: 8DF81F81-019F-348E-B47E-40A4484B9E21
Kernel slide:     0x000000000fe00000
Kernel text base: 0xffffff8010000000
__HIB  text base: 0xffffff800ff00000
System model name: MacBookPro12,1 (Mac-E43C1C25D4880AD6)

System uptime in nanoseconds: 41768246106524
last loaded kext at 38178201321305: com.apple.driver.AppleXsanScheme	3 (addr 0xffffff7f9320d000, size 32768)
last unloaded kext at 38770256206523: com.apple.driver.AppleXsanScheme	3 (addr 0xffffff7f9320d000, size 32768)
loaded kexts:
com.github.kbfuse.filesystems.kbfuse	3.5.2
org.virtualbox.kext.VBoxNetAdp	5.1.8
org.virtualbox.kext.VBoxNetFlt	5.1.8
org.virtualbox.kext.VBoxUSB	5.1.8
org.virtualbox.kext.VBoxDrv	5.1.8
com.intel.kext.intelhaxm	1.1.1
com.apple.driver.AppleBluetoothMultitouch	92
com.apple.filesystems.smbfs	3.1.1
com.apple.driver.AudioAUUC	1.70
com.apple.driver.AGPM	110.23.14
com.apple.driver.X86PlatformShim	1.0.0
com.apple.driver.ApplePlatformEnabler	2.7.0d0
com.apple.filesystems.autofs	3.0
com.apple.driver.AppleOSXWatchdog	1
com.apple.driver.AppleGraphicsDevicePolicy	3.13.78
com.apple.driver.AppleUpstreamUserClient	3.6.4
com.apple.driver.AppleHDA	278.56
com.apple.driver.AppleIntelBDWGraphics	10.2.2
com.apple.driver.pmtelemetry	1
com.apple.iokit.IOUserEthernet	1.0.1
com.apple.driver.AppleLPC	3.1
com.apple.iokit.IOBluetoothSerialManager	5.0.2f4
com.apple.Dont_Steal_Mac_OS_X	7.0.0
com.apple.driver.AppleHV	1
com.apple.driver.AppleThunderboltIP	3.0.8
com.apple.iokit.BroadcomBluetoothHostControllerUSBTransport	5.0.2f4
com.apple.driver.AppleCameraInterface	5.57.0
com.apple.driver.AppleSMCLMU	208
com.apple.driver.AppleIntelSlowAdaptiveClocking	4.0.0
com.apple.driver.AppleBacklight	170.9.10
com.apple.driver.AppleMCCSControl	1.2.15
com.apple.driver.AppleIntelBDWGraphicsFramebuffer	10.2.2
com.apple.driver.AppleUSBCardReader	404.30.1
com.apple.AppleFSCompression.AppleFSCompressionTypeDataless	1.0.0d1
com.apple.AppleFSCompression.AppleFSCompressionTypeZlib	1.0.0
com.apple.BootCache	40
com.apple.filesystems.hfs.kext	366.30.3
com.apple.iokit.IOAHCIBlockStorage	295.20.1
com.apple.driver.AppleAHCIPort	326
com.apple.driver.AppleTopCaseHIDEventDriver	104
com.apple.driver.AirPort.Brcm4360	1130.3.1a4
com.apple.driver.AppleSmartBatteryManager	161.0.0
com.apple.driver.AppleRTC	2.0
com.apple.driver.AppleACPIButtons	5.0
com.apple.driver.AppleHPET	1.8
com.apple.driver.AppleSMBIOS	2.1
com.apple.driver.AppleACPIEC	5.0
com.apple.driver.AppleAPIC	1.7
com.apple.nke.applicationfirewall	172
com.apple.security.quarantine	3
com.apple.security.TMSafetyNet	8
com.apple.driver.usb.cdc.ecm	5.0.0
com.apple.driver.usb.cdc	5.0.0
com.apple.driver.usb.AppleUSBHostCompositeDevice	1.1
com.apple.driver.AppleBluetoothHIDKeyboard	197
com.apple.driver.IOBluetoothHIDDriver	5.0.2f4
com.apple.iokit.IOUSBUserClient	900.4.1
com.apple.kext.triggers	1.0
com.apple.driver.DspFuncLib	278.56
com.apple.kext.OSvKernDSPLib	525
com.apple.driver.X86PlatformPlugin	1.0.0
com.apple.driver.IOPlatformPluginFamily	6.0.0d8
com.apple.iokit.IOSurface	153.3
com.apple.driver.AppleSSE	1.0
com.apple.driver.AppleGraphicsControl	3.13.78
com.apple.iokit.IOBluetoothHostControllerUSBTransport	5.0.2f4
com.apple.iokit.IOBluetoothHostControllerTransport	5.0.2f4
com.apple.iokit.IOBluetoothFamily	5.0.2f4
com.apple.driver.AppleHDAController	278.56
com.apple.iokit.IOHDAFamily	278.56
com.apple.iokit.IOAudioFamily	205.11
com.apple.vecLib.kext	1.2.0
com.apple.iokit.IOSerialFamily	11
com.apple.iokit.IOSlowAdaptiveClockingFamily	1.0.0
com.apple.driver.AppleBacklightExpert	1.1.0
com.apple.iokit.IONDRVSupport	513.1
com.apple.driver.AppleSMC	3.1.9
com.apple.driver.AppleSMBusController	1.0.14d1
com.apple.AppleGraphicsDeviceControl	3.13.78
com.apple.iokit.IOAcceleratorFamily2	289.27
com.apple.iokit.IOGraphicsFamily	513.1
com.apple.iokit.IOSCSIBlockCommandsDevice	394.30.2
com.apple.iokit.IOUSBMassStorageDriver	131.1.1
com.apple.iokit.IOSCSIArchitectureModelFamily	394.30.2
com.apple.driver.usb.networking	5.0.0
com.apple.driver.CoreStorage	540.30.1
com.apple.filesystems.hfs.encodings.kext	1
com.apple.driver.AppleThunderboltDPInAdapter	4.6.1
com.apple.driver.AppleThunderboltDPAdapterFamily	4.6.1
com.apple.driver.AppleThunderboltPCIDownAdapter	2.1.1
com.apple.iokit.IOAHCIFamily	288
com.apple.driver.AppleHIDKeyboard	197
com.apple.driver.AppleMultitouchDriver	368.7
com.apple.driver.AppleInputDeviceSupport	76.4
com.apple.driver.AppleHSSPIHIDDriver	49
com.apple.driver.AppleThunderboltNHI	4.1.8
com.apple.iokit.IOThunderboltFamily	6.3.3
com.apple.iokit.IO80211Family	1200.12.2
com.apple.driver.mDNSOffloadUserClient	1.0.1b8
com.apple.iokit.IONetworkingFamily	3.2
com.apple.driver.corecapture	1.0.4
com.apple.driver.AppleHSSPISupport	49
com.apple.driver.usb.AppleUSBHostPacketFilter	1.0
com.apple.iokit.IOUSBFamily	900.4.1
com.apple.driver.AppleUSBHostMergeProperties	1.1
com.apple.driver.AppleIntelLpssSpiController	3.0.60
com.apple.driver.AppleIntelLpssGspi	3.0.60
com.apple.driver.AppleIntelLpssDmac	3.0.60
com.apple.driver.usb.AppleUSBXHCIPCI	1.1
com.apple.driver.usb.AppleUSBXHCI	1.1
com.apple.driver.AppleEFINVRAM	2.1
com.apple.driver.AppleEFIRuntime	2.0
com.apple.iokit.IOHIDFamily	2.0.0
com.apple.iokit.IOSMBusFamily	1.1
com.apple.security.sandbox	300.0
com.apple.kext.AppleMatch	1.0.0d1
com.apple.driver.AppleKeyStore	2
com.apple.driver.AppleMobileFileIntegrity	1.0.5
com.apple.driver.AppleCredentialManager	1.0
com.apple.driver.KernelRelayHost	1
com.apple.iokit.IOUSBHostFamily	1.1
com.apple.driver.AppleBusPowerController	1.0
com.apple.driver.DiskImages	444.30.5
com.apple.iokit.IOStorageFamily	2.1
com.apple.iokit.IOReportFamily	31
com.apple.driver.AppleFDEKeyStore	28.30
com.apple.driver.AppleACPIPlatform	5.0
com.apple.iokit.IOPCIFamily	2.9
com.apple.iokit.IOACPIFamily	1.4
com.apple.kec.Libm	1
com.apple.kec.pthread	1
com.apple.kec.corecrypto	1.0
```

#### Yesterday
```
Anonymous UUID:       6EC9C959-1935-64E3-89E8-47EFC08F4DDA

Wed Dec 21 19:29:57 2016

*** Panic Report ***
panic(cpu 1 caller 0xffffff800dde568d): "TLB invalidation IPI timeout: " "CPU(s) failed to respond to interrupts, unresponsive CPU bitmap: 0xc, NMIPI acks: orig: 0x0, now: 0x0"@/Library/Caches/com.apple.xbs/Sources/xnu/xnu-3789.31.2/osfmk/x86_64/pmap.c:2724
Backtrace (CPU 1), Frame : Return Address
0xffffff81f6a83a90 : 0xffffff800dcf211c 
0xffffff81f6a83b10 : 0xffffff800dde568d 
0xffffff81f6a83bb0 : 0xffffff800ddec7a2 
0xffffff81f6a83cd0 : 0xffffff800dded7da 
0xffffff81f6a83d40 : 0xffffff800dd7a7cb 
0xffffff81f6a83e80 : 0xffffff800dd79abc 
0xffffff81f6a83eb0 : 0xffffff800dd75623 
0xffffff81f6a83ee0 : 0xffffff800dd3de9d 
0xffffff81f6a83f90 : 0xffffff800dd9ba27 
0xffffff81f6a83fb0 : 0xffffff800dca0af7 

BSD process name corresponding to current thread: kernel_task

Mac OS version:
16C67

Kernel version:
Darwin Kernel Version 16.3.0: Thu Nov 17 20:23:58 PST 2016; root:xnu-3789.31.2~1/RELEASE_X86_64
Kernel UUID: 8DF81F81-019F-348E-B47E-40A4484B9E21
Kernel slide:     0x000000000da00000
Kernel text base: 0xffffff800dc00000
__HIB  text base: 0xffffff800db00000
System model name: MacBookPro12,1 (Mac-E43C1C25D4880AD6)

System uptime in nanoseconds: 185155142878849
last loaded kext at 181455235053707: com.apple.driver.usb.cdc	5.0.0 (addr 0xffffff7f90d6d000, size 28672)
last unloaded kext at 181692235370933: com.apple.driver.usb.AppleUSBHostCompositeDevice	1.1 (addr 0xffffff7f90d66000, size 28672)
loaded kexts:
com.github.kbfuse.filesystems.kbfuse	3.5.2
org.virtualbox.kext.VBoxNetAdp	5.1.8
org.virtualbox.kext.VBoxNetFlt	5.1.8
org.virtualbox.kext.VBoxUSB	5.1.8
org.virtualbox.kext.VBoxDrv	5.1.8
com.intel.kext.intelhaxm	1.1.1
com.apple.filesystems.smbfs	3.1.1
com.apple.iokit.BroadcomBluetoothHostControllerUSBTransport	5.0.2f4
com.apple.filesystems.autofs	3.0
com.apple.driver.AudioAUUC	1.70
com.apple.driver.ApplePlatformEnabler	2.7.0d0
com.apple.driver.AGPM	110.23.14
com.apple.driver.X86PlatformShim	1.0.0
com.apple.driver.pmtelemetry	1
com.apple.iokit.IOUserEthernet	1.0.1
com.apple.iokit.IOBluetoothSerialManager	5.0.2f4
com.apple.Dont_Steal_Mac_OS_X	7.0.0
com.apple.driver.AppleGraphicsDevicePolicy	3.13.78
com.apple.driver.AppleUpstreamUserClient	3.6.4
com.apple.driver.AppleOSXWatchdog	1
com.apple.driver.AppleHDA	278.56
com.apple.driver.AppleIntelBDWGraphics	10.2.2
com.apple.driver.AppleCameraInterface	5.57.0
com.apple.driver.AppleThunderboltIP	3.0.8
com.apple.driver.AppleLPC	3.1
com.apple.driver.AppleSMCLMU	208
com.apple.driver.AppleHV	1
com.apple.driver.AppleIntelSlowAdaptiveClocking	4.0.0
com.apple.driver.AppleIntelBDWGraphicsFramebuffer	10.2.2
com.apple.driver.AppleBacklight	170.9.10
com.apple.driver.AppleMCCSControl	1.2.15
com.apple.driver.AppleUSBCardReader	404.30.1
com.apple.AppleFSCompression.AppleFSCompressionTypeDataless	1.0.0d1
com.apple.AppleFSCompression.AppleFSCompressionTypeZlib	1.0.0
com.apple.BootCache	40
com.apple.filesystems.hfs.kext	366.30.3
com.apple.iokit.IOAHCIBlockStorage	295.20.1
com.apple.driver.AppleAHCIPort	326
com.apple.driver.AppleTopCaseHIDEventDriver	104
com.apple.driver.AirPort.Brcm4360	1130.3.1a4
com.apple.driver.AppleSmartBatteryManager	161.0.0
com.apple.driver.AppleACPIButtons	5.0
com.apple.driver.AppleRTC	2.0
com.apple.driver.AppleHPET	1.8
com.apple.driver.AppleSMBIOS	2.1
com.apple.driver.AppleACPIEC	5.0
com.apple.driver.AppleAPIC	1.7
com.apple.nke.applicationfirewall	172
com.apple.security.quarantine	3
com.apple.security.TMSafetyNet	8
com.apple.iokit.IOBluetoothHostControllerUSBTransport	5.0.2f4
com.apple.iokit.IOBluetoothHostControllerTransport	5.0.2f4
com.apple.iokit.IOUSBUserClient	900.4.1
com.apple.kext.triggers	1.0
com.apple.iokit.IOBluetoothFamily	5.0.2f4
com.apple.driver.AppleSSE	1.0
com.apple.driver.DspFuncLib	278.56
com.apple.kext.OSvKernDSPLib	525
com.apple.iokit.IOSurface	153.3
com.apple.driver.AppleGraphicsControl	3.13.78
com.apple.driver.AppleHDAController	278.56
com.apple.iokit.IOHDAFamily	278.56
com.apple.iokit.IOAudioFamily	205.11
com.apple.vecLib.kext	1.2.0
com.apple.driver.X86PlatformPlugin	1.0.0
com.apple.driver.IOPlatformPluginFamily	6.0.0d8
com.apple.iokit.IOSerialFamily	11
com.apple.iokit.IOSlowAdaptiveClockingFamily	1.0.0
com.apple.AppleGraphicsDeviceControl	3.13.78
com.apple.iokit.IOAcceleratorFamily2	289.27
com.apple.driver.AppleBacklightExpert	1.1.0
com.apple.iokit.IONDRVSupport	513.1
com.apple.driver.AppleSMC	3.1.9
com.apple.driver.AppleSMBusController	1.0.14d1
com.apple.iokit.IOGraphicsFamily	513.1
com.apple.iokit.IOSCSIBlockCommandsDevice	394.30.2
com.apple.iokit.IOUSBMassStorageDriver	131.1.1
com.apple.iokit.IOSCSIArchitectureModelFamily	394.30.2
com.apple.driver.usb.networking	5.0.0
com.apple.driver.CoreStorage	540.30.1
com.apple.filesystems.hfs.encodings.kext	1
com.apple.iokit.IOAHCIFamily	288
com.apple.driver.AppleThunderboltDPInAdapter	4.6.1
com.apple.driver.AppleThunderboltDPAdapterFamily	4.6.1
com.apple.driver.AppleThunderboltPCIDownAdapter	2.1.1
com.apple.driver.AppleHIDKeyboard	197
com.apple.driver.AppleMultitouchDriver	368.7
com.apple.driver.AppleInputDeviceSupport	76.4
com.apple.driver.AppleHSSPIHIDDriver	49
com.apple.driver.AppleThunderboltNHI	4.1.8
com.apple.iokit.IOThunderboltFamily	6.3.3
com.apple.iokit.IO80211Family	1200.12.2
com.apple.driver.mDNSOffloadUserClient	1.0.1b8
com.apple.iokit.IONetworkingFamily	3.2
com.apple.driver.corecapture	1.0.4
com.apple.driver.AppleHSSPISupport	49
com.apple.driver.AppleIntelLpssSpiController	3.0.60
com.apple.driver.AppleIntelLpssGspi	3.0.60
com.apple.driver.AppleIntelLpssDmac	3.0.60
com.apple.driver.usb.AppleUSBXHCIPCI	1.1
com.apple.driver.usb.AppleUSBXHCI	1.1
com.apple.driver.usb.AppleUSBHostPacketFilter	1.0
com.apple.iokit.IOUSBFamily	900.4.1
com.apple.driver.AppleUSBHostMergeProperties	1.1
com.apple.driver.AppleEFINVRAM	2.1
com.apple.driver.AppleEFIRuntime	2.0
com.apple.iokit.IOHIDFamily	2.0.0
com.apple.iokit.IOSMBusFamily	1.1
com.apple.security.sandbox	300.0
com.apple.kext.AppleMatch	1.0.0d1
com.apple.driver.AppleKeyStore	2
com.apple.driver.AppleMobileFileIntegrity	1.0.5
com.apple.driver.AppleCredentialManager	1.0
com.apple.driver.KernelRelayHost	1
com.apple.iokit.IOUSBHostFamily	1.1
com.apple.driver.AppleBusPowerController	1.0
com.apple.driver.DiskImages	444.30.5
com.apple.iokit.IOStorageFamily	2.1
com.apple.iokit.IOReportFamily	31
com.apple.driver.AppleFDEKeyStore	28.30
com.apple.driver.AppleACPIPlatform	5.0
com.apple.iokit.IOPCIFamily	2.9
com.apple.iokit.IOACPIFamily	1.4
com.apple.kec.Libm	1
com.apple.kec.pthread	1
com.apple.kec.corecrypto	1.0

System Profile:
Graphics: Intel Iris Graphics 6100, Intel Iris Graphics 6100, Built-In
AirPort: spairport_wireless_card_type_airport_extreme (0x14E4, 0x133), Broadcom BCM43xx 1.0 (7.21.171.68.1a4)
Bluetooth: Version 5.0.2f4, 3 services, 27 devices, 1 incoming serial ports
PCI Card: Apple 57762-A0, Ethernet Controller, Thunderbolt@195,0,0
Thunderbolt Bus: MacBook Pro, Apple Inc., 27.1
Thunderbolt Device: Thunderbolt to Gigabit Ethernet Adapter, Apple Inc., 3, 5.5
Memory Module: BANK 0/DIMM0, 8 GB, DDR3, 1867 MHz, 0x02FE, 0x4544464232333241314D412D4A442D460000
Memory Module: BANK 1/DIMM0, 8 GB, DDR3, 1867 MHz, 0x02FE, 0x4544464232333241314D412D4A442D460000
USB Device: USB 3.0 Bus
USB Device: Card Reader
USB Device: Bluetooth USB Host Controller
Serial ATA Device: APPLE SSD SM0128G, 121.33 GB
Network Service: Thunderbolt Ethernet, Ethernet, en4
Network Service: Wi-Fi, AirPort, en0
Model: MacBookPro12,1, BootROM MBP121.0167.B18, 2 processors, Intel Core i7, 3.1 GHz, 16 GB, SMC 2.28f7
```
</p></details>

**Comment by dsheets** [comment]
@johanneswuerbach it sounds like the first issue was a Moby Linux (Docker for Mac's VM distro) kernel panic but it looks like the second issue is a macOS kernel panic. Is that right?

Could you please open a new issue with the details of your host kernel panic? Thanks!
